os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -466,7 +502,8 @@ static void recovery_user_assert(void)
 		if (g_faultmsg_sender && (msg = (faultmsg_t *)sq_remfirst(&g_freemsg_list))) {
 			msg->binid = binid;
 			sq_addlast((sq_entry_t *)msg, (sq_queue_t *)&g_faultmsg_list);
-			up_unblock_task(g_faultmsg_sender);
+			/* Unblock fault message sender */
+			recovery_unblock_task(g_faultmsg_sender);
```

Comment:
What's the meaning of this line? Does `recovery` mean the module name? or the verb?

Reply:
I suggest `xx_activate_faultmsg_sender` without giving argument.
The g_faultmsg_sender is the global variable even this function also accesses here and called function is the same file. You don't need to give the address from here.

Reply:
Is it possible for the fault message sender to prepare a message containing bin_id?
It might reduce the fault handling time of the fault handler a bit.